parent/pom.xml

```diff
@@ -78,6 +78,7 @@
     <version.netty>4.1.117.Final</version.netty>
     <version.objenesis>3.4</version.objenesis>
     <version.opensearch>2.14.0</version.opensearch>
```

Comment:
❓️What is this version used for now then?

Reply:
this is used for "opensearch-rest-client". "opensearch-java" has a direct dependency on it.
For example opensearch-java:2.19.0 depends on opensearch-rest-client:2.12.0